src/index.html

```diff
@@ -22,6 +22,371 @@
+            class="nav__item"
+            data-qa="nav-hover"
+          >
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Laptops &amp; Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="product"
+          class="card__image"
+        />
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rates">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rates-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </button>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="product"
+          class="card__image"
+        />
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__rates">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rates-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__button">buy</button>
```

Comment:
The button element is missing the `type` attribute. As mentioned in the previous comment, it's a good practice to define the button type explicitly to avoid unexpected behaviors.